tools/templates/legion-component/component.styles.js

```diff
@@ -0,0 +1,16 @@
+
+module.exports = (componentName) => ({
+    content: `import {
+        I${componentName}Styles
+    } from './${componentName}.types';
+    import { IProcessedStyleSet, mergeStyleSets } from '@fluentui/react';
+
+    export const getStyles = (): IProcessedStyleSet<I${componentName}Styles> => {
+        return mergeStyleSets({
```

Comment:
how do you style fluent controls in this pattern? Like when you waht to change a combo box or button style, can you still use the nested properties with strong types?

Reply:
I haven't been able to replicate this behavior, even with the classnames. I need to dig deeper to find out how, however as a temporary solution we could define the styles prop value for a subcomponent in the styles file.